i18n/zh-Hans/docusaurus-plugin-content-docs/current/academy/academy-intro/academy-overview.md

```diff
@@ -27,13 +27,13 @@ A set of specific learning objectives is provided at the beginning of the lesson
 
 ## Content
 
-The core content in each lesson is conveyed through structured, short format videos followed by scripts. This hybrid visual and text approach ensures accessibility and caters to diverse learning preferences.
+The core content in each lesson is conveyed through structured, short format videos followed by scripts. This hybrid visual and text approach ensures accessibility and caters to diverse learning preferences. This hybrid visual and text approach ensures accessibility and caters to diverse learning preferences.
```

Comment:
I am seeing a number of these duplicated strings and some other formatting errors. 
Please hold off on merging this pr as I further review